plasmapy/formulary/relativity.py

```diff
@@ -88,3 +88,54 @@ def Lorentz_factor(V: u.m / u.s):
             gamma = ((1 - (V / c) ** 2) ** -0.5).value
 
     return gamma
+
+
+@validate_quantities(V={"can_be_negative": True}, m={"can_be_negative": False})
+def relativistic_energy(V: u.m / u.s, m: u.kg):
```

Comment:
```suggestion
@validate_quantities(m={"can_be_negative": False}, 
                     validations_on_return={"can_be_negative": False})
def relativistic_energy(V: u.m / u.s, m: u.kg) -> u.Joule:
```


1. You don't need to define `"can_be_negative": True` for `V` since that is the default value.
1. add units and validations for the return value